centreon/www/front_src/src/Dashboards/SingleInstancePage/Dashboard/AddEditWidget/WidgetProperties/Inputs/Metrics/useMetrics.ts

```diff
@@ -303,6 +305,15 @@ const useMetrics = (propertyName: string): UseMetricsOnlyState => {
     useDeepCompare([servicesMetrics, resources])
   );
 
+  useEffect(() => {
+    const services = map(
+      pick(['uuid', 'id', 'name', 'parentName']),
+      servicesMetrics?.result || []
+    );
+
+    setFieldValue(`data.services`, services);
+  }, [values?.data?.[propertyName]]);
+
```

Comment:
```suggestion
```
Why? There are no occurence of the data.services field value

Reply:
@Thebarda  we need this information for the link to RS page from the single metric widget, to display the graph panel on RS page